symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/impl/BotAuthenticatorCertImpl.java

```diff
@@ -0,0 +1,73 @@
+package com.symphony.bdk.core.auth.impl;
+
+import com.symphony.bdk.core.api.invoker.ApiClient;
+import com.symphony.bdk.core.api.invoker.ApiException;
+import com.symphony.bdk.core.api.invoker.ApiRuntimeException;
+import com.symphony.bdk.core.auth.AuthSession;
+import com.symphony.bdk.core.auth.BotAuthenticator;
+import com.symphony.bdk.core.auth.exception.AuthUnauthorizedException;
+import com.symphony.bdk.gen.api.CertificateAuthenticationApi;
+import com.symphony.bdk.gen.api.model.Token;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apiguardian.api.API;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Bot authenticator certificate implementation.
+ *
+ * @see
+ * <a href="https://developers.symphony.com/symphony-developer/docs/bot-authentication-workflow-1">Bot
+ * Authentication Workflow</a>
```

Comment:
Weird behaviour if the formatter, it could be written on a single line. Do we know if the Javadoc is still valid even if written like this?

Reply:
I guess so but max_line_length is set to 100 in the .editorconfig file which is quite small (IntelliJ default value is 120)

Reply:
While I find it nice to automatically wrap long lines I'd say for comments it is up to the writer to make it readable
Often it breaks URLs
I found "Wrap at right margin" that could work for javadoc comments
![image](https://user-images.githubusercontent.com/63661676/91272557-33d85d00-e77c-11ea-99f1-51f85b2505f8.png)

Reply:
In .editorconfig if I use 
_ij_java_wrap_comments = false
ij_java_wrap_long_lines = false_

it seems to not wrap comments (but it still wraps code)

Reply:
(the 100 margin comes from SBE I think, I agree that we could probably move to 120)